src/core/blockchain/cosmos/config.ts

```diff
@@ -127,7 +127,8 @@ export const config: IBlockchainConfig = {
     ui: {
         validator: {
             totalLabel: 'Validator.totalDelegated',
-            amountCardLabel: 'Validator.delegation'
+            amountCardLabel: 'Validator.delegation',
+            maximumNumberOfValidators: 5
```

Comment:
there is no such thing on cosmos, so you can leave it like this, and use `0` as value, or make the property optional.